services/motion/builtin/builtin_test.go

```diff
@@ -170,29 +170,26 @@ func TestMoveWithObstacles(t *testing.T) {
 }
 
 func TestMoveSingleComponent(t *testing.T) {
-	t.Skip()
+	ms, teardown := setupMotionServiceFromConfig(t, "../data/moving_arm.json")
+	defer teardown()
+
+	grabPose := spatialmath.NewPoseFromPoint(r3.Vector{-25, 30, 0})
 	t.Run("succeeds when all frame info in config", func(t *testing.T) {
-		ms, teardown := setupMotionServiceFromConfig(t, "../data/moving_arm.json")
-		defer teardown()
-		grabPose := referenceframe.NewPoseInFrame("c", spatialmath.NewPoseFromPoint(r3.Vector{-25, 30, 0}))
 		_, err := ms.MoveSingleComponent(
 			context.Background(),
 			arm.Named("pieceArm"),
-			grabPose,
+			referenceframe.NewPoseInFrame("c", grabPose),
```

Comment:
[q]: this is very interesting! It seems that we are doing the same thing but slightly differently.
we went from:
```
grabPose := referenceframe.NewPoseInFrame("c", spatialmath.NewPoseFromPoint(r3.Vector{-25, 30, 0}))
```

to 
```
grabPose := spatialmath.NewPoseFromPoint(r3.Vector{-25, 30, 0})
referenceframe.NewPoseInFrame("c", grabPose)
````

how does this prevent the flake from re-occuring?